website/src/pages/index.js

```diff
@@ -200,6 +200,7 @@ function Home() {
                     alt={quote.name}
                     className="avatar__photo avatar__photo--xl"
                     img={quote.thumbnail}
+                    style={{overflow: 'hidden'}}
```

Comment:
I have a task to fix this in Infima.

Reply:
oops my bad. lets remove this after uve finished patching infima :)